docs/core/reference/cored.md

```diff
@@ -2,14 +2,14 @@
 
 Command `cored` provides the Chain Core daemon and API server.
 
-## Installation
-
-`cored` is included with all desktop installations of Chain Core
+`cored` is included with all official releases of Chain Core
 Developer Edition, and is run by default when starting Mac, Windows and Docker.
 
-### From source
+If you plan to make changes to the internals of Chain Core, you can find
+build instructions in our [GitHub readme](https://github.com/chain/chain/blob/main/Readme.md#building-from-source);
+if you want to _use_ chain core, please download one of our official releases.
 
-_The instructions in this section require having the Go programming environment installed and the PATH variable correctly configured. See the [Chain Core Readme file](https://github.com/chain/chain/blob/main/Readme.md#building-from-source) for details._
+### Build tags
```

Comment:
So if we're going to refer people to the readme to build it, would it be possible to omit any mention of the build tags in this doc?

I would slightly prefer to describe the behavior differences between developer edition and the server build in terms of user-visible things, e.g. "Chain Core Developer Edition has some features designed for convenience during development and testing: it has a reset button to delete all data, it permits unauthenticated requests over the loopback device (localhost), it has a mock HSM, and it allows unencrypted (non-TLS) HTTP requests. These features are unsafe for use in production and are disabled in Chain Core Enterprise Edition."

(I don't know if "Chain Core Enterprise Edition" is exactly the right name for that build, but you get the idea.)